src/LodeRunner.API.Test/IntegrationTests/Extensions/TestRunPayloadExtension.cs

```diff
@@ -18,25 +18,85 @@ public static class TestRunPayloadExtension
         /// <param name="testRunPayload">TestRunPayload.</param>
         /// <param name="name">TestRun name.</param>
         public static void SetMockData(this TestRunPayload testRunPayload, string name)
+        {
+            testRunPayload.SetNameAndTime(name);
+
+            testRunPayload.LoadTestConfig = CreateLoadTestConfig();
+
+            testRunPayload.LoadClients = CreateLoadClients(2);
+        }
+
+        /// <summary>
+        /// Sets the mock data to load test LodeRunner API.
+        /// </summary>
+        /// <param name="testRunPayload">The test run payload.</param>
+        /// <param name="name">The name.</param>
+        /// <param name="clientStatusId">The clientStatusId.</param>
+        /// <param name="apiPort">The apiPort.</param>
+        /// <param name="loadClientCount">The number of loadClients to include in Test Run.</param>
+        public static void SetMockDataToLoadTestLodeRunnerApi(this TestRunPayload testRunPayload, string name, string clientStatusId, int apiPort, int loadClientCount = 1)
+        {
+            testRunPayload.SetNameAndTime(name);
+
+            var loadTestConfig = new LoadTestConfig()
+            {
+                Name = $"Sample LoadTestConfig - IntegrationTesting-{DateTime.UtcNow:yyyy'-'MM'-'dd'T'HH':'mm':'ss.fffffffK}",
+                Files = new List<string>() { "LodeRunner.Api-benchmark.json" },
+                Server = new List<string>() { $"http://localhost:{apiPort}" },
+            };
+
+            testRunPayload.LoadTestConfig = loadTestConfig;
+
+            testRunPayload.LoadClients = CreateLoadClients(loadClientCount, clientStatusId);

```

Comment:
Could CreateLoadClients cause weird stuff to happen if the loadClientCount is greater than 0 and clientStatusId is not null or empty?

Reply:
good catch , refactored method to only create 1 LoadClient if clientStatusis is not Null or empty.